### PR TITLE
python

### PR DESCRIPTION
**Pull request by BotellaA**

### CODE REVIEW

File: pages/how-to-use-binding.vue [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    <ul :class="paragraph">
+      <li>3.6 and 3.7 on Windows - Visual Studio 2017</li>
+      <li>3.6, 3.7 and 3.8 on Ubuntu 18.04</li>
+      <li>2.7 on RHEL 7 / CentOS 7</li>
+      <li>3.6, 3.7 and 3.8 on macOS X Catalina (10.15)</li>
+    </ul>
+
+    <p :class="paragraph">
+      Pre-compiled binaries on these three platforms are also provided.
```
Comment (panquez) [inline pages/how-to-use-binding.vue]: three?

File: pages/how-to-use-binding.vue [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+    <h3 :class="subsection">
+      On Windows
+    </h3>
+
+    <h4 :class="subsubsection">
+      Configuration
+    </h4>
+
+    <ol :class="paragraph">
+      <li>The bindgins are located in the bin directory of the package.</li>
```
Comment (panquez) [inline pages/how-to-use-binding.vue]:
```suggestion
      <li>The bindings are located in the bin directory of the package.</li>
```

File: pages/how-to-use-binding.vue [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+      <li>Add this path to the <code>PYHTONPATH</code> and the <code>PATH</code> environment variables.</li>
+    </ol>
+
+    <h4 :class="subsubsection">
+      Execution
+    </h4>
+
+    <ol :class="paragraph">
+      <li>Open a Python script or Jupyter notebook.</li>
+      <li>Import OpenGeode bindings, there is one bindings for each C++ library. Example for the basic library:
```
Comment (panquez) [inline pages/how-to-use-binding.vue]:
```suggestion
      <li>Import OpenGeode bindings, there is one binding for each C++ library. Example for the basic library: 
```

File: pages/how-to-use-binding.vue [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    <h3 :class="subsection">
+      On Linux and macOS
+    </h3>
+
+    <h4 :class="subsubsection">
+      Configuration
+    </h4>
+
+    <ol :class="paragraph">
+      <li>The bindgins are located in the lib directory of the package.</li>
```
Comment (panquez) [inline pages/how-to-use-binding.vue]:
```suggestion
      <li>The bindings are located in the lib directory of the package.</li>
```

File: pages/how-to-use-binding.vue [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+      <li>Add this path to the <code>PYHTONPATH</code> and the <code>LD_LIBRARY_PATH</code> environment variables.</li>
+    </ol>
+
+    <h4 :class="subsubsection">
+      Execution
+    </h4>
+
+    <ol :class="paragraph">
+      <li>Open a Python script or Jupyter notebook.</li>
+      <li>Import OpenGeode bindings, there is one bindings for each C++ library. Example for the basic library:
```
Comment (panquez) [inline pages/how-to-use-binding.vue]:
```suggestion
      <li>Import OpenGeode bindings, there is one binding for each C++ library. Example for the basic library: 
```

**Review by panquez — APPROVED** [review]
Few comments but then it is OK for me

File: pages/how-to-use-binding.vue (hunk 1 above)
Comment (BotellaA) [reply pages/how-to-use-binding.vue]: yes!

**Pull request merged**